http-api/src/lib.rs

```diff
@@ -251,6 +260,34 @@ async fn readme_handler(
     Ok(warp::reply::json(&blob))
 }
 
+/// List all projects
+async fn project_root_handler(ctx: Context) -> Result<Json, Rejection> {
+    use radicle_daemon::git::identities::SomeIdentity;
+    #[derive(serde::Serialize)]
+    struct Project {
+        name: String,
+        urn: String,
+    }
+
+    let storage = ReadOnly::open(&ctx.paths).unwrap();
+    let projects = identities::any::list(&storage).unwrap()
```

Comment:
These errors should be propagated like in the other handlers.

Reply:
Fixed https://github.com/radicle-dev/radicle-client-services/pull/75/commits/4e629abc369bc3d005f87bbd99a7dfb65987b450#diff-191ca8c6d151a1a4c5243319a8395c8a17c03c6e257bf4d796b004f19a0bf469L272-R276